tests/test-tag-and-attribute-sanitizer.php

```diff
@@ -302,7 +302,10 @@ function () {
 									</amp-img>
 								</amp-story-grid-layer>
 								<amp-story-grid-layer template="vertical">
-									<h1>The End</h1>
+									<h1 animate-in="drop" animate-in-delay="500ms" animate-in-duration="600ms">The End</h1>
+									<div>
```

Comment:
Wouldn't that have the `amp-story-block-wrapper` class name now?

Reply:
I haven't updated the tests yet, also, templates need update.